test/e2e/specs/site-editor/template-parts-mode.spec.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.describe( 'Template Parts for Classic themes', () => {
+	test.beforeAll( async ( { requestUtils } ) => {
+		await requestUtils.activateTheme( 'emptyhybrid' );
+	} );
+
+	test.afterAll( async ( { requestUtils } ) => {
+		await requestUtils.activateTheme( 'twentytwentyone' );
+	} );
+
+	test( 'can access template parts list page', async ( { admin, page } ) => {
+		await admin.visitAdminPage(
+			'site-editor.php',
+			'postType=wp_template_part&path=/wp_template_part/all'
+		);
```

Comment:
Nit: Could this be changed to `visitSiteEditor`?

Reply:
I tried that, but the `visitSiteEditor` checks for the canvas, and the list component isn't rendered inside the iframe.